rpcsx-gpu/CMakeLists.txt

```diff
@@ -1,9 +1,16 @@
+find_package(Vulkan REQUIRED)
+find_package(glfw3 3.3 REQUIRED)
 
 add_executable(rpcsx-gpu
   main.cpp
 )
 
-target_include_directories(rpcsx-gpu PUBLIC .)
-target_link_libraries(rpcsx-gpu PUBLIC amdgpu::bridge amdgpu::device glfw vulkan)
+target_include_directories(rpcsx-gpu
+    PUBLIC
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
We don't need binary_dir, just source_dir

Reply:
This more of a future proof as it's of no real effect until you start to generate or configure files since they are generated into the binary dir by default.

Reply:
rpcsx-gpu and rpcsx-os will not generate anything. If we need to generate something, there will be another target for it with public include dir. Your change has very real effect for my IDE - I need to scroll all those generated directories to pick header file for include.